processor/k8sprocessor/config.go

```diff
@@ -173,3 +175,15 @@ type FieldFilterConfig struct {
 	//   equals, not-equals, exists, does-not-exist.
 	Op string `mapstructure:"op"`
 }
+
+// PodAssociationConfig allows specifying rules for associating resources
```

Comment:
Why are we mentioning "resources" here? Does "resource" here have the meaning of the "Resource" as OpenTelemetry's understands it?